test/create_game_section/Lobby/lobby_widget_test.dart

```diff
@@ -0,0 +1,69 @@
+import 'dart:async';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:niira/main.dart';
+import 'package:niira/models/user_data.dart';
+import 'package:niira/services/auth/auth_service.dart';
+
+final UserData mockUser = UserData(
```

Comment:
I think we have multiple UserData mocks scattered around now - maybe make a factory class in the mocks folder?